### PR TITLE
kernel: bump 6.1 to 6.1.81

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.81

All patches automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, ramips/tplink_archer-a6-v3, flogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: x86/64/AMD Cezanne, ramips/tplink_archer-a6-v3, flogic/xiaomi_redmi-router-ax6000-ubootmod

### CODE REVIEW

**Review by graysky2 — COMMENTED** [review]
If I recursive grep for `CONFIG_EFI_EARLYCON` within the `targets/linux/` There are entries for several others.  I do not know if I should add the new symbol to them or not.
```
armsr/config-6.1
d1/config-6.1
sifiveu/config-6.1
x86/64/config-6.1
x86/generic/config-6.1
```

EDIT: bah, just moved it to generic

File: target/linux/generic/config-6.1 [hunk 1]
```diff
@@ -1954,6 +1954,7 @@ CONFIG_DVB_MAX_ADAPTERS=16
 # CONFIG_EEPROM_LEGACY is not set
 # CONFIG_EEPROM_MAX6875 is not set
 # CONFIG_EFI is not set
+CONFIG_EFI_HANDOVER_PROTOCOL=y
```
Comment (DragonBluep) [inline target/linux/generic/config-6.1:1957]: This symbol is only for ARCH x86. I guess we need to move it to x86 sub-target.

Comment (graysky2) [reply target/linux/generic/config-6.1]: I moved it to `target/linux/x86/config-6.1`

**Comment by schuettecarsten** [comment]
Tested on TP-Link Archer C7 v5 (ath79), Belkin RT3200 (mt7622) and RPi4 (bcm27xx). No issues here.

**Pull request merged**